lib/scuba/wrapper.js

```diff
@@ -28,6 +29,8 @@ class ScubaClientImpl extends ScubaClient {
             if (!this.enabled) {
                 this._log.info('Scuba health check passed, enabling quotas');
             }
+            monitoring.utilizationServiceHealth.labels({ state: 'healthy' }).set(1);
+            monitoring.utilizationServiceHealth.labels({ state: 'unhealthy' }).set(0);
```

Comment:
avoid multiple metrics like this does not seem very "user friendly": a single `available` metric with a pseuod-boolean value (0 = no, 1 = yes), like the `up` metrics, seems a more standard approach...

Reply:
Now we have no label https://github.com/scality/cloudserver/pull/5556/files#diff-7a879f848f4d512bac1250da6f152d89b8feab003de8d9fc6fa202cb6b17dcf5R34